gradio/cli/commands/components/dev.py

```diff
@@ -63,6 +63,18 @@ def _dev(
         "gradio", gradio_path, cli_arg_name="--gradio-path"
     )
 
+    gradio_node_path = subprocess.Popen(
+        [node, "-e", "console.log(require.resolve('@gradio/preview'))"],
+        cwd=Path(component_directory / "frontend"),
+        stdout=subprocess.PIPE,
+    )
+
+    if gradio_node_path.stdout is None:
+        raise ValueError(
```

Comment:
We should not raise an error here because custom component developers developing old component will be blocked.

Reply:
We need to raise an error here because the location of the files needed to run dev mode changed.